public/503_database_not_ready.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>SUSE CaaS Platform is initializing</title>
+  <title>Platform is initializing</title>
```

Comment:
Maybe we can put `Platform still initializing` instead of `is` - I feel it sounds better?

Reply:
i guess we can safely drop the static pages/assets from velum entirely, as we have them in `velum-branding` so we dont need to duplicate them

Reply:
I would use `Platform is being initialized`